### PR TITLE
logged in formatting changed to fit rest of header

### LINKED ISSUE #769: Fix styling of the Header once user is logged in
Currently, there are some elements on the header that are not properly styled. They should be fixed for a better user experience. 

image

### PR DESCRIPTION
**Pull request by agarcia-caicedo**
## Issue This PR Addresses
fixes #769, incorporates some of #770

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

This will fix the formatting for the logged in view of the header. This change also follows a better material-ui style approach.

The loggedout component was also changed, as I used it for some testing and thought it would be better to push it as well, rather than delete it and have someone else re-do it.

Because the logged in view redirects to the dev site, the preview branch won't display the change.

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by cindyledev** [comment]
Tested this on Zeit and it's a little bit buggy because when you go to 
https://telescope-5d6aeh670.now.sh/ to preview and sign in using the credentials, you get redirected to https://dev.telescope.cdot.systems/

It shows that you're signed in as such
Annotation 2020-03-03 023842
Annotation 2020-03-03 023842_

You would have to navigate back to https://telescope-5d6aeh670.now.sh/ and hit refresh to see
Annotation 2020-03-03 023842__

**Comment by cindyledev** [comment]
Tested it locally as well and it works as it should
- Signed into `localhost:3000` and got redirected to `localhost:3000` with state as signed in

Deleted cookies to sign out then I tested it again
- Signed into `localhost:8000` and got redirected to `localhost:3000` with state as signed in
^ I think this is what's suppose to happen for `localhost:8000`?

**Comment by agarcia-caicedo** [comment]
> But yes, the styling can be revisited in the future. The main point of this PR was to follow the MUI approach and it's nice to be able to see the 'welcome user' instead of trying to read the tiny barely visible font size

Ths styling was the main reason for the pull request and it looks like logout button is not being styled at all

File: src/frontend/src/components/Login/LoggedIn.jsx [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
 function LoggedIn(props) {
   const { telescopeUrl } = useSiteMetadata();
   const logoutUrl = `${telescopeUrl}/auth/logout`;
+  const classes = useStyles();
 
   return (
     <div>
-      Welcome {props.email} <a href={logoutUrl}> Logout </a>
+      <Button className={classes.button}>
+        <a href={logoutUrl}> Logout </a>
```
Comment (cindyledev) [inline src/frontend/src/components/Login/LoggedIn.jsx]: I think you just forgot classes.link here

Comment (agarcia-caicedo) [reply src/frontend/src/components/Login/LoggedIn.jsx]: thanks

File: src/frontend/src/components/Login/Login.css [hunk 2]
```diff
@@ -1,10 +1,3 @@
 .avatar {
```
Comment (humphd) [inline src/frontend/src/components/Login/Login.css]: Can we move the `avatar` styles into your JS as well and delete this file?

Comment (agarcia-caicedo) [reply src/frontend/src/components/Login/Login.css]: Changes made

File: src/frontend/src/components/Login/LoggedIn.jsx [hunk 3]
```diff
@@ -1,15 +1,38 @@
...
   return (
     <div>
-      Welcome {props.email} <a href={logoutUrl}> Logout </a>
+      <Button className={classes.button}>
+        <a href={logoutUrl} className={classes.link}>
+          Logout
+        </a>
+      </Button>
+      <Typography variant="h4">| Welcome {props.email}</Typography>
       <img className="avatar" src={`https://unavatar.now.sh/${props.email}`} alt={props.email} />
```
Comment (humphd) [inline src/frontend/src/components/Login/LoggedIn.jsx]: `className={classes.avatar}`

**Pull request merged**